yarn.lock

```diff
@@ -1069,12 +1069,12 @@
     "@types/yargs" "^15.0.0"
     chalk "^4.0.0"
 
-"@ministryofjustice/frontend@0.0.17-alpha":
-  version "0.0.17-alpha"
-  resolved "https://registry.yarnpkg.com/@ministryofjustice/frontend/-/frontend-0.0.17-alpha.tgz#a5e2e2d2dce58102e9245153bd6d4992e7977c5b"
-  integrity sha512-UOwqFA24kd8xJY6XrWxaQxM2xzr8BHdwb5L82xPk3rG7hgEnWRppk7vt9eszqSt3Hwdfj63pD8aaMTMsi89P4Q==
+"@ministryofjustice/frontend@^0.0.19-alpha":
+  version "0.0.19-alpha"
+  resolved "https://registry.yarnpkg.com/@ministryofjustice/frontend/-/frontend-0.0.19-alpha.tgz#4f00cb02cafef3f8cbfa9d4665885fa766309cdf"
+  integrity sha512-iNNJSIr+poRiBWxyoi4pxs2qcYRfFzVO9yh6ALPUeKDj9pCm1lWYReWhr02/jLgLEJ3nKdJDxIbGv0c/RxSR/w==
   dependencies:
-    moment "^2.22.2"
+    moment "^2.26.0"
```

Comment:
I just noticed this depends on moment... I get the feeling our JS bundle might be a lot bigger than it needs to be. (70kb! https://bundlephobia.com/result?p=moment@2.26.0)